app/store/index.ts

```diff
@@ -1,18 +1,15 @@
 import AsyncStorage from '@react-native-community/async-storage';
 import { createImmutableStateInvariantMiddleware } from '@reduxjs/toolkit';
-import { applyMiddleware, compose, createStore } from 'redux';
+import { applyMiddleware, createStore } from 'redux';
 import { createMigrate, persistReducer, persistStore } from 'redux-persist';
+import thunk from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
 
 import migrations from './migrations';
 import rootReducer from './reducers/rootReducer';
 
-const composeEnhancers =
-  // eslint-disable-next-line no-underscore-dangle
-  (typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) ||
-  compose;
-
-const enhancers = composeEnhancers(
-  applyMiddleware(createImmutableStateInvariantMiddleware()),
+const enhancers = composeWithDevTools(
+  applyMiddleware(thunk, createImmutableStateInvariantMiddleware()),
```

Comment:
This is the exact same thing, but removes the headache of us manually typing the `window` global for typescript